### ISSUE TITLE
Query logic - set email as lowercase BEFORE it reaches the query

### ISSUE THREAD

**Issue by lucasjkr** [lucasjkr/opencommerce#34]
Many queries search email by saying LOWER(email), when MySQL should find that regardless. Verify, and if that specificity isn't needed, remove from the queries.

NOTE: there should still be an attempt to normalize this, but it should be PHP converting the string to lower BEFORE it gets inserted or searched for.

**Comment by arnisjuraga** [comment]
can be done in Validiation?
https://github.com/lucasjkr/opencommerce/issues/32#issuecomment-334802049

**Comment by lucasjkr** [comment]
Yes! I guess a validation class, first method is email, either returns the email address in lowercase or false.
